packages/batch-poster-monitor/index.ts

```diff
@@ -232,27 +232,89 @@ const getBatchPosterLowBalanceAlertMessage = async (
       )
     } catch {
       // batchPoster not found by any means
-      return 'Batch poster information not found'
+      throw Error('Batch poster information not found')
     }
   }
 
-  const balance = await parentChainClient.getBalance({
+  return batchPoster
+}
+
+const getBatchPosterLowBalanceAlertMessage = async (
+  parentChainClient: PublicClient,
+  childChainInformation: ChainInfo,
+  sequencerInboxLogs: EventLogs
+) => {
+  if (sequencerInboxLogs.length === 0) return null
```

Comment:
Should we default to something here?
For the case where there hasn't been any batches posted because the batch poster doesn't have enough balance. In those cases, I think we don't report a low balance in the batch poster in the current implementation.

Reply:
Added a static low balance check -  if sequencer inbox logs are not found to calculate dynamic-balance-threshold, we will revert to checking the current balance with a static value of 0.1 ETH (customizable)

Reply:
Nice!!